projects/core/components/dialog/dialog.service.ts

```diff
@@ -1,24 +1,24 @@
-import {Injectable} from '@angular/core';
-import {AbstractTuiDialogService} from '@taiga-ui/cdk';
+import {Injectable, Inject, inject} from '@angular/core';
+import {AbstractTuiDialogService, TuiIdService} from '@taiga-ui/cdk';
 import {TuiDialogOptions} from '@taiga-ui/core/interfaces';
+import {TUI_DIALOG_OPTIONS} from '@taiga-ui/core/tokens/dialog-options';
 import {PolymorpheusComponent} from '@tinkoff/ng-polymorpheus';
 
 import {TuiDialogComponent} from './dialog.component';
 
 const DIALOG = new PolymorpheusComponent(TuiDialogComponent);
-const DEFAULT_OPTIONS = {
-    size: `m`,
-    required: false,
-    closeable: true,
-    dismissible: true,
-    label: ``,
-    header: ``,
-} as const;
 
 @Injectable({
     providedIn: `root`,
 })
 export class TuiDialogService extends AbstractTuiDialogService<TuiDialogOptions<any>> {
     protected readonly component = DIALOG;
-    protected readonly defaultOptions: TuiDialogOptions<any> = DEFAULT_OPTIONS as any;
+    protected readonly defaultOptions: TuiDialogOptions<any> = {
+        ...inject(TUI_DIALOG_OPTIONS),
+        data: undefined,
+    };
+
+    constructor(@Inject(TuiIdService) idService: TuiIdService) {
```

Comment:
Is this constructor required? 🤔

Reply:
Can you please check the comment I made above

Reply:
Got it, thank you.